CODE_OF_CONDUCT.md

```diff
@@ -33,7 +33,7 @@ that we ask all community members to adhere to:
 -   be friendly and patient,
 -   be considerate,
 -   be kind,
--   be careful in the words that you choose,
+-   be careful in the words that we choose,
 -   when we disagree, try to understand why, and
```

Comment:
I think the "you to we" changes introduce way too much ambiguity. It's never clear whether "we" is referring to users, contributors or even the lead carbon team. Look below as an example where "we" is referring to two separate entities.